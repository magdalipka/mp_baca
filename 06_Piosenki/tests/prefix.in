3
7
aaa aab aac aaa aa aa a
6
ILoveMyFriends1 ILoveMyFriends2 ILoveMyFriends3 ILoveMyFriends4 ILoveMyFriends5 ILoveMyFriends
5
Silent Silent Silent Silent Silent
