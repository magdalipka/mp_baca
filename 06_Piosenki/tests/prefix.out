aaa aa aab aa aac a aaa
a
ILoveMyFriends1 ILoveMyFriends4 ILoveMyFriends2 ILoveMyFriends5 ILoveMyFriends3 ILoveMyFriends
ILoveMyFriends
Silent Silent Silent Silent Silent
Silent
